docs/development/standards.rst

```diff
@@ -158,6 +161,10 @@ save some work while typing docker compose commands. This section explains these
        you can run ``inv docker.attach web`` and jump into a pdb session
        (it also works with ipdb and pdb++)
 
+    .. tip::
+
+       You can hit CTRL-p CTRL-p to detach it without stopping the running process.
```

Comment:
Huh, I didn't know that! :)

Reply:
It's kind of new: https://github.com/readthedocs/common/pull/81

We work on the shadows :smile: